src/leader_scheduler.rs

```diff
@@ -18,11 +18,15 @@ use solana_sdk::vote_transaction::VoteTransaction;
 use std::io::Cursor;
 use std::sync::Arc;
 
-pub const TICKS_PER_BLOCK: u64 = 4;
-pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = TICKS_PER_BLOCK * 256;
-pub const DEFAULT_LEADER_ROTATION_INTERVAL: u64 = TICKS_PER_BLOCK * 32;
-pub const DEFAULT_SEED_ROTATION_INTERVAL: u64 = TICKS_PER_BLOCK * 256;
-pub const DEFAULT_ACTIVE_WINDOW_LENGTH: u64 = TICKS_PER_BLOCK * 256;
+// At 10 ticks/s, 8 ticks per slot implies that leader rotation and voting will happen
+// every 800 ms. A fast voting cadence ensures faster finality and convergence
+pub const DEFAULT_TICKS_PER_SLOT: u64 = 8;
+// Bootstrap height lasts for ~100 seconds
+pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = 1024;
+// The number of slots per epoch.
```

Comment:
nit: remove this comment ("The number of slots per epoch.") as that really adds nothing over the name of the constant that yells the same thing at you: `DEFAULT_SLOTS_PER_EPOCH`